components/src/core/drawer/drawer.component.ts

```diff
@@ -41,6 +41,7 @@ export class DrawerComponent extends StateListener implements OnInit, OnChanges
     @Input() open: boolean;
     @Input() condensed = false;
     @Input() sideBorder = false;
+    @Input() enableSelectionHierarchy = true;
```

Comment:
can we make this more descriptive? Like `disableActiveItemParentStyles` or something like that? Should is be disable____ instead of enable______ since the default is on?

Reply:
I'm happy with the name `disableActiveItemParentStyles `, I'll update accordingly.

Reply:
This is updated now.